### DISCUSSION TITLE
最近的更新状态

### DISCUSSION THREAD

**Discussion by linonetwo** [rubickCenter/rubick#156] [General]
因为觉得居然能用 Electron 做到这么多事情很酷，所以我过去点watch关注了这个项目，之前有段时间经常收到推送。不过我之前一直没有用这种框的需求，就下了一直没打开过…

今天是感觉经常查单词和向 GPT 问题不太方便，于是就下载了一个 Cerebro 。结果想解决一下他无法快速对选中的文本做操作的问题，就搜索了「electron 获取用户选中的文本」，搜到了介绍rubick的文章 《基于 Electron 实现 uTools 的超级面板》，于是就想起原来之前还关注过这个应用，直接下下来用了。

不过看到最近几个月都没怎么更新，感觉奇怪，遂发此问关心一下。

**Comment by Seryta** [comment]
同问，想在linux上用，但是看最近没更新了
